client.lua

```diff
@@ -89,11 +91,64 @@ local function hasHarness(items)
     harness = _harness
 end
 
+-- Weapon Stress [Haroki]
+local function IsWhitelistedWeaponStress(weapon)
+    if weapon then
+        for _, v in pairs(config.WhitelistedWeaponStress) do
+            if weapon == v then
+                return true
+            end
+        end
+    end
+    return false
+end
+
+local hasWeapon = false
+
+local function startWeaponStressThread(weapon)
+    if IsWhitelistedWeaponStress(weapon) then hasWeapon = false; return end
+
+    CreateThread(function()
+        while hasWeapon do
+            local ped = cache.ped
+            if IsPedShooting(ped) then
```

Comment:
```suggestion
            if IsPedShooting(cache.ped) then
```
I don't feel like another variable is really needed here.